CHANGES.rst

```diff
@@ -7,7 +7,17 @@ Changes
 `Unreleased <https://github.com/Ouranosinc/cowbird/tree/master>`_ (latest)
 ------------------------------------------------------------------------------------
 
-* Nothing yet.
+Features / Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+* Add synchronization of Magpie permissions between different services, when receiving incoming webhooks.
```

Comment:
I will specify it is `Magpie services` we are talking about here.

I put the refactoring of `services` to `handlers` in a TODO ( I will create an issue too ), because services is used everywhere in the code, and I didn't want to add even more to this PR that is already too big.

Reply:
You could also add a note like this : 
```
(relates to `#22 <https://github.com/Ouranosinc/cowbird/issues/22>`_)
```

This way it will be clear what the mentioned confusion to be wary of refers to.